frontend/modelarchive/src/main/java/com/amazonaws/ml/mms/archive/LegacyManifest.java

```diff
@@ -113,7 +113,7 @@ public Manifest migrate() throws InvalidModelException {
             Object engineVersion = engine.get("MXNet");
             if (engineVersion instanceof Number) {
                 Manifest.Engine eng = new Manifest.Engine();
-                eng.setEngineName(Manifest.EngineType.MX_NET);
+                eng.setEngineName("MXNet");
```

Comment:
You don't need change it to String, the only the you need change is:
if (engineType.value.equals(value)) {

change it to:
if (engineType.value.equalsIgnoreCase(value)) {

Reply:
I made these changes because MMS doesn't use Engine and this gives flexibility for customers to define the name that they want to use. Also, this removes MMS from needing to make code changes when ever we add support for newer frameworks. I will drop by and discuss this in person.